doc/scenario.md

```diff
@@ -281,15 +279,33 @@ List of facts:
   - `instance_dist_dir` - path to instance link to distributed package;
 - `single_instances_for_each_machine` - list of instances (Ansible hosts), one for each physical machine. Can be used,
   for example, in `delegate_to`;
-- `not_expelled_instance` - information about one not expelled instance. It's a dictionary with fields:
-  - `name` - instance name (Ansible host);
-  - `console_sock` - path to control socket of instance;
 - `scenario_steps` - description of scenario steps. Each step is a dictionary with fields:
   - `name` - name of step;
   - `path` - path to YAML file of step;
-- `control_instance` - information about control instance (will be set if `cartridge_control_instance` is specified);
-- `delivered_package_path` - remote path to file of delivered package
-  (will be set if `cartridge_delivered_package_path` is specified).
+
+There are other facts that are set during role execution or intialized with values
```

Comment:
What about `package_info` e.t.c. These are facts that are also established in the process of performing the role.